ncm-spma/src/main/pan/components/spma/dnf/schema.pan

```diff
@@ -22,4 +22,14 @@ type component_spma_dnf = {
     "run"                ? boolean  # Run the transaction after configuring DNF
 };
 
+@{DNF module structure. To be used as dict, with name as key.
+ to set default profiles for modules not enabled by default and change existing defaults.
+}
+type component_spma_dnf_module_defaults = {
+    'stream'   : string
+    'profiles' ?  string[]
```

Comment:
I'm assuming neither of these should have any leading or trailing white-space?
```suggestion
    'stream'   : string_trimmed
    'profiles' ?  string_trimmed[]
```

Reply:
interesting, our release doesn't seem to have this, updated anyway, we can look at this internally.